src/en/index.html

```diff
@@ -13,7 +13,7 @@ <h1 class="h1 visually-hidden">Ceph.io {{ title }}</h1>
   <div class="wrapper">
     <div class="color-white max-w-192 relative z-1">
       <h2 class="h1 mb-8">The Future of Storage<sup>™</sup></h2>
-      <p class="mb-10 standout">Ceph is an open source distributed storage system designed to evolve with&nbsp;data.</p>
+      <p class="mb-10 standout">Ceph is an open-source, distributed storage system.</p>
```

Comment:
``Ceph is an open source distributed storage system designed to scale with your data``?